### PR TITLE
Fix typos in lib/public subdirectory

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S l10n -L jus ./lib/private`

Signed-off-by: luz paz <luzpaz@github.com>

### CODE REVIEW

**Review by PVince81 — APPROVED** [review]
:+1: looks fine

File: lib/public/Accounts/IAccount.php [hunk 1]
```diff
@@ -135,7 +135,7 @@ public function getAllProperties(): Generator;
 	public function setPropertyCollection(IAccountPropertyCollection $propertyCollection): IAccount;
 
 	/**
-	 * Returns the requestes propery collection (multi-value properties)
+	 * Returns the requests property collection (multi-value properties)
```
Comment (come-nc) [inline lib/public/Accounts/IAccount.php:138]:
```suggestion
	 * Returns the requested property collection (multi-value properties)
```

**Review by come-nc — CHANGES_REQUESTED** [review]

**Comment by luzpaz** [comment]
Squashed. Shall I also rebase or not necessary?

**Review by come-nc — APPROVED** [review]

**Comment by come-nc** [comment]
> Squashed. Shall I also rebase or not necessary?

I do not think a rebase is needed, but the DCO bot is somehow unhappy, can you look into it?

**Comment by luzpaz** [comment]
Not sure why.

**Comment by szaimen** [comment]
I think it expects a different mail but this is not a problem and we can just set it to skip, afaics.

**Review by skjnldsv — APPROVED** [review]

**Comment by skjnldsv** [comment]
Please fix conflict and we can merge

**Pull request merged**